### PR TITLE
Define separate resource params, remove unused floorist params

### PR DESCRIPTION
**Pull request by bsquizz**
Separates CPU/mem requests & limits into separate parameters for each app component -- allowing us to tune them for better resource utilization in ephemeral environments.

### CODE REVIEW

**Review by thearifismail — DISMISSED** [review]
LGTM

File: deploy/clowdapp.yml [hunk 1]
```diff
@@ -704,30 +704,79 @@ objects:
...
+- name: CPU_LIMIT_PENDO_SYNCHER
+  value: 500m
+- name: MEMORY_REQUEST_PENDO_SYNCHER
   value: 256Mi
-- description: request limit for reaper
-  name: MEMORY_REQUEST_REAPER
+- name: MEMORY_LIMIT_PENDO_SYNCHER
+  value: 512Mi
+
+- name: CPU_LIMIT_SYNCHRONIZER
```
Comment (kruai) [inline deploy/clowdapp.yml:771]:
```suggestion
- name: CPU_REQUEST_SYNCHRONIZER
```

File: deploy/clowdapp.yml [hunk 2]
```diff
@@ -704,30 +704,79 @@ objects:
...
+- name: CPU_REQUEST_MQ_SP
+  value: 250m
+- name: CPU_LIMIT_MQ_SP
+  value: 500m
+- name: MEMORY_REQUEST_MQ_SP
+  value: 256Mi
+- name: MEMORY_LIMIT_MQ_SP
+  value: 512Mi
+
+- name: CPU_LIMIT_REAPER
```
Comment (kruai) [inline deploy/clowdapp.yml:744]:
```suggestion
- name: CPU_REQUEST_REAPER
```

File: deploy/clowdapp.yml [hunk 3]
```diff
@@ -704,30 +704,79 @@ objects:
...
+  value: 256Mi
+- name: MEMORY_LIMIT_REAPER
   value: 512Mi
-- description: memory limit of host-synchronizer
-  name: MEMORY_LIMIT_SYNC
-  value: 3Gi
-- description: memory limit of reaper
-  name: MEMORY_LIMIT_REAPER
+
+- name: CPU_LIMIT_SP_VALIDATOR
```
Comment (kruai) [inline deploy/clowdapp.yml:753]:
```suggestion
- name: CPU_REQUEST_SP_VALIDATOR
```

File: deploy/clowdapp.yml [hunk 4]
```diff
@@ -704,30 +704,79 @@ objects:
...
+  value: 500m
+- name: MEMORY_REQUEST_SP_VALIDATOR
+  value: 256Mi
+- name: MEMORY_LIMIT_SP_VALIDATOR
   value: 512Mi
-- description: request limit for service
-  name: MEMORY_REQUEST
+
+- name: CPU_LIMIT_PENDO_SYNCHER
```
Comment (kruai) [inline deploy/clowdapp.yml:762]:
```suggestion
- name: CPU_REQUEST_PENDO_SYNCHER
```

**Review by kruai — CHANGES_REQUESTED** [review]
The title says it's a draft, but the PR isn't a draft. Should it be?

File: deploy/clowdapp.yml (hunk 1 above)
Comment (bsquizz) [reply deploy/clowdapp.yml]: Good catches here, fixed all of these

**Comment by bsquizz** [comment]
Changes were merged into app-interface to support these new param names.

**Comment by kruai** [comment]
@bsquizz Looks like the values in deploy-clowder.yml in app-interface don't abide by the limit-to-request ratio restriction (i.e. can't be more than 2:1)

**Review by kruai — APPROVED** [review]
LGTM

**Pull request merged**